src/openagent/experts/defi_expert.py

```diff
@@ -0,0 +1,92 @@
+from typing import Optional, Type
+
+from langchain.callbacks.manager import AsyncCallbackManagerForToolRun
+from langchain.tools import BaseTool
+from loguru import logger
+from pydantic import BaseModel, Field
+from rss3_dsl_sdk.client import RSS3Client
```

Comment:
add poetry dependency

Reply:
used 'poetry add xxx' to add those packages